### ISSUE TITLE
CID22: full dataset annotations not included in download

### ISSUE THREAD

**Issue by lilith** [cloudinary/ssimulacra2#17]
The CID22 full dataset download (7.2 GB, 250 references, ~22k compressed images) only includes `CID22_validation_set.csv` with annotations for the 49-reference validation subset (4,292 scored pairs).

The remaining ~17.7k pairs from the other 201 references have no published human scores. The [CID22 labs page](https://cloudinary.com/labs/cid22) doesn't mention that the full set annotations are withheld — it describes the dataset as containing "22,000 subjective scores" but only ~4,300 are in the download.

Could the full set annotations (MCOS/RMOS/Elo scores for all 250 references) be published? This would make CID22 much more useful as a benchmark for independent metric development and comparison.

**Comment by jonsneyers** [comment]
This is what the labs page says:
> The full CID22 dataset consists of two parts:
> - a training set based on 201 pristine images, and
> - a validation set based on 49 pristine images.
>
> The annotations for the validation set (mean bias-corrected opinion scores) are freely available to the research community.

This does imply that the annotations for the remaining 201 references are not currently made publicly available.

Possibly at some point in the future the full dataset will be released, but as of today, only the validation set has been released publicly.

**Comment by lilith** [comment]
Ah. Thank you. It's unfortunate the references aren't available. I [made a highly optimized Rust implementation of ssimulacra2](https://github.com/imazen/fast-ssim2) that is about 3x faster than the prior one and a [Rust Butteraugli implementation that is 2-3x faster than the C++ one](https://github.com/imazen/butteraugli) and was able to [backport a 20% perf boost to libjxl](https://github.com/libjxl/libjxl/pull/4749). I then ported both to the GPU with Rust and managed to make them very, *very* fast, but the CUDA drivers struggle hard under that sustained load and crash my system after a few hours and a few hundred thousand images. 

Either way, I need a metric an order of magnitude faster than either - on CPU - to embed in my codecs for a efficient targeting loop. I've made various pareto frontier advances with jpeg and avif (and some slight ones with my Rust implementation of libjxl), but I recently realized real-world gains are more often about pinpointing that subperceptual configuration per-image per-viewing-condition. I've developed a [pretty decent proxy](https://github.com/imazen/zensim) for ssim2, but the TID2013 and KADID-10k datasets don't inspire a ton of confidence for codec development. 

I've done some massive synthetic data generation that after many moons has reached an okay point, but I think CID22 would really do a better job.  

I'm probably going to need to figure out and run a human score study soon unless I can find more data sets. My current research is showing lots of promise around driving encoder configuration decisions with a shallow MLP where input=(with target metric, time bounds, and a quickly-extracted feature set from images). One-shot results are startling good(analyze+mlp = ~4ms) and make the metric-loop super short, but the metric still needs to be fast.

**Comment by jonsneyers** [comment]
The AIC-4 project that is currently ongoing in the JPEG committee will soon be producing more subjective ground truth. You might be interested in joining this activity. See https://jpeg.org/aic/index.html
The AIC adhoc group is open for anyone to participate.
